modules/cassandra/src/main/java/org/testcontainers/containers/CassandraContainer.java

```diff
@@ -166,14 +178,23 @@ public String getPassword() {
     }
 
     /**
-     * Get configured Cluster
+     * Get configured Session
```

Comment:
Why?

```suggestion
     * Get configured Cluster
```

Reply:
There is no more `Cluster` vs `Session` distinction in driver4. Your only entry point is the session, now called `CqlSession`. The word "cluster" becomes meaningless.

Reply:
this only applies to v4, while in v3 (the currently used types that we're deprecating) is remains `Cluster`.

Reply:
Whoops you are right, my bad.

Reply:
Yes, one of my first commits for this changed `getCluster()` to `getSession()`, but that was reverted. I just forgot to revert the comment change.